### PR TITLE
feat(runtime): Introduce `PerformanceMultiplier` as `Config` constant

### PR DESCRIPTION
**Pull request by DennisInSky**
@reviewer-or-team

### CODE REVIEW

File: pallets/gear-debug/src/mock.rs [hunk 1]
```diff
@@ -150,6 +151,7 @@ impl pallet_gear::Config for Test {
     type GasPrice = GasConverter;
     type WeightInfo = ();
     type OutgoingLimit = OutgoingLimit;
+    type CostMultiplier = ConstMultiplier;
```
Comment (NikVolf) [inline pallets/gear-debug/src/mock.rs:154]: is it cost or const

Comment (DennisInSky) [reply pallets/gear-debug/src/mock.rs]: It is `cost` for the time being, so this is a typo. The actual/proper name for this syscall/functionality is not known yet. Any ideas?

File: gcore/src/exec.rs [hunk 2]
```diff
@@ -80,6 +80,29 @@ pub fn block_timestamp() -> u64 {
     timestamp
 }
 
+/// Get current cost multiplier.
+///
+/// The cost multiplier is a value that is used to calculate the cost of
+/// BLA-BLA-BLA.
```
Comment (DennisInSky) [inline gcore/src/exec.rs:86]: Some description is needed here. So far I can't express what this syscall is for

File: pallets/gear-debug/src/mock.rs (hunk 1 above)
Comment (NikVolf) [reply pallets/gear-debug/src/mock.rs]: Reopen in once it is known then

File: gcore/src/exec.rs (hunk 2 above)
Comment (NikVolf) [reply gcore/src/exec.rs:86]: Any updates on this research?

File: core/src/percent.rs [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+// along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+//! Basic struct for working with integer percentages.
+
+use core::cmp::Ord;
+use num_traits::{cast::NumCast, Num};
+use parity_scale_codec::{Decode, Encode};
+use scale_info::TypeInfo;
+
+/// Basic struct for working with integer percentages.
```
Comment (breathx) [inline core/src/percent.rs:26]: Consider: You don't need this type in core. It's enough to keep just u32 here but introduce `Percent` in `gcore/gstd` in future

**Review by breathx — APPROVED** [review]

**Review by shamilsan — APPROVED** [review]

**Pull request merged**